tests/wpt/web-platform-tests/common/get-host-info.sub.js

```diff
@@ -4,7 +4,7 @@ function get_host_info() {
   var HTTP_PORT2 = '{{ports[http][1]}}';
   var HTTPS_PORT = '{{ports[https][0]}}';
   var ORIGINAL_HOST = '{{host}}';
-  var REMOTE_HOST = (ORIGINAL_HOST === 'localhost') ? '127.0.0.1' : ('www1.' + ORIGINAL_HOST);
+  var REMOTE_HOST = (ORIGINAL_HOST === '127.0.0.1') ? '127.0.0.2' : '127.0.0.1';
```

Comment:
@jgraham the reason for this change is to run the test suite with a non-similar `REMOTE_HOST`. At the moment, `ORIGINAL_HOST` and `REMOTE_HOST` are similar-origin, so we never actually run any tests cross-thread.